### PR TITLE
Collapse back to upstream reconcile command

### PR DESCRIPTION
**Pull request by deads2k**
Reverts the change that depends on internal types (kubectl is quickly moving away from those).  This will fail until https://bugzilla.redhat.com/show_bug.cgi?id=1597334 is done.

@sdodson @openshift/sig-master

### CODE REVIEW

**Comment by soltysh** [comment]
/hold
@deads2k you need to fix the title for the upstream revert b/c it's failing travis

**Comment by deads2k** [comment]
title fixed

/hold cancel

**Comment by soltysh** [comment]
@deads2k still not quite right for the commit message, you need `UPSTREAM: revert: <sha>: <normal upstream format>` to pass through it. `UPSTREAM: revert: <drop>` you have here won't work.

**Comment by deads2k** [comment]
> @deads2k still not quite right for the commit message, you need UPSTREAM: revert: <sha>: <normal upstream format> to pass through it. UPSTREAM: revert: <drop> you have here won't work.

I'll fix the script.  With the fork and the bot, we no longer need that level of specificity.

File: vendor/k8s.io/kubernetes/pkg/kubectl/cmd/auth/reconcile.go [hunk 1]
```diff
@@ -23,16 +23,18 @@ import (
...
 	rbacv1client "k8s.io/client-go/kubernetes/typed/rbac/v1"
-	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
 	"k8s.io/kubernetes/pkg/kubectl/genericclioptions/printers"
 	"k8s.io/kubernetes/pkg/kubectl/genericclioptions/resource"
 	"k8s.io/kubernetes/pkg/kubectl/scheme"
 	"k8s.io/kubernetes/pkg/registry/rbac/reconciliation"
+	"fmt"
```
Comment (tnozicka) [inline vendor/k8s.io/kubernetes/pkg/kubectl/cmd/auth/reconcile.go]: nit: reorder to the top with "errors"

**Pull request merged**